spec/features/allocate_feature_spec.rb

```diff
@@ -143,25 +153,29 @@
     expect(page).to have_content('This reason cannot be more than 175 characters')
   end
 
-  scenario 're-allocating', vcr: { cassette_name: 'prison_api/re_allocate_feature' } do
+  scenario 're-allocating', :js, vcr: { cassette_name: 'prison_api/re_allocate_feature' } do
     create(
       :allocation_history,
       prison: 'LEI',
       nomis_offender_id: nomis_offender_id,
       primary_pom_nomis_id: 485_735,
       recommended_pom_type: 'probation'
     )
-
+    # 'Goes to allocations' page
```

Comment:
Minor niggle: is the `'` quotation mark in the wrong place in this comment?

```suggestion
    # Goes to 'allocations' page
```